content/en/docs/v3.3/learning/learner.md

```diff
@@ -70,7 +70,7 @@ Learner only serves as a standby node until promoted: Leadership cannot be trans
 
 In addition, etcd limits the total number of learners that a cluster can have, and avoids overloading the leader with log replication. Learner never promotes itself. While etcd provides learner status information and safety checks, cluster operator must make the final decision whether to promote learner or not.
 
-## Features in v3.5
+## Proposed Features in Future Releases
```

Comment:
I'd suggest using sentence-case for the heading (in this and other pages):

> ## Proposed features for future releases

Reply:
makes sense

Reply:
@chalin updated.